### DISCUSSION TITLE
Can a maintainer add this to conda-forge?

### DISCUSSION THREAD

**Discussion by ckoppelman** [microsoft/presidio#697] [Requested features] [answered]
Hi!

I'd love to use `presidio_analyzer` in my conda project, but I can't find it in a conda provider.  Can a maintainer who will keep it to-date please [add this to conda-forge](https://conda-forge.org/#add_recipe)?  (this is a dupe of [this issue](https://github.com/microsoft/presidio/issues/698))

Thanks!

Charles

**Comment by omri374** [comment, marked as answer]
Hi @ckoppelman, thanks for the suggestion! We'll add this to our backlog.
